src/styles/fonts.sss

```diff
@@ -26,4 +26,3 @@
     src:
         url("../fonts/Source_Sans_Pro/SourceSansPro-SemiBold.ttf")
         format("truetype")
-
```

Comment:
Please configure your editor to not remove the last newline in a file, it creates unnecessary whitespace diff.